hyperspace/core/src/command.rs

```diff
@@ -111,19 +112,22 @@ impl Cmd {
 			any_chain_b.name(),
 			client_id_a_on_b
 		);
-		Ok(())
+		config.chain_a.set_client_id(client_id_b_on_a);
+		config.chain_b.set_client_id(client_id_a_on_b);
```

Comment:
```suggestion
		config.chain_b.set_client_id(client_id_b_on_a);
```

Reply:
oh I didn't see this one - fixed in [924c218](https://github.com/ComposableFi/centauri/pull/145/commits/924c218e65def7953f0cfe1c1ae6032e9a45b307)